src/constants/earthEngineLayers/__tests__/earthEngineLayers.spec.js

```diff
@@ -0,0 +1,198 @@
+            minItems: 1,
+        },
+        band: { type: 'string' },
+        bands: {
+            type: 'object',
+            required: ['label', 'multiple', 'list'],
+            properties: {
+                label: { type: 'string' },
+                multiple: { type: 'boolean' },
+                default: { type: 'string' },
+                list: {
+                    type: 'array',
+                    items: {
+                        type: 'object',
+                        required: ['id', 'name'],
+                        properties: {
+                            id: { type: 'string' },
+                            name: { type: 'string' },
+                        },
+                    },
+                    minItems: 1,
+                },
+            },
+        },
+        methods: {
+            type: ['array', 'object'],
+            items: {
+                type: 'object',
+                required: ['name', 'arguments'],
+                properties: {
+                    name: {
+                        type: 'string',
+                        enum: ['multiply', 'toFloat', 'subtract'],
+                    },
+                    arguments: { type: 'array' },
+                },
+            },
+            minItems: 1,
+            properties: {
+                multiply: {
+                    type: 'array',
+                    items: { type: 'number' },
+                },
+            },
+        },
+        style: {
+            type: ['array', 'object'],
+            items: {
+                type: 'object',
+                required: ['value', 'name', 'color'],
+                properties: {
+                    value: { type: 'number' },
+                    name: { type: 'string' },
+                    color: { type: 'string' },
+                },
+            },
+            minItems: 1,
+            properties: {
+                color: { type: 'string' },
+                strokeWidth: { type: 'number' },
+                min: { type: 'number' },
+                max: { type: 'number' },
+                palette: {
+                    type: 'array',
+                    items: { type: 'string' },
+                    minItems: 1,
+                },
+            },
+        },
+        popup: { type: 'string' },
+        maskOperator: { type: 'string', enum: ['gte', 'gt'] },
+        opacity: { type: 'number' },
+        mosaic: { type: 'boolean' },
+        tileScale: { type: 'number' },
+    },
+}
+
+const ajv = new Ajv()
+const validateConfig = (config) => {
+    const validate = ajv.compile(configSchema)
+    const valid = validate(config)
+    if (!valid) {
+        throw new Error(
+            JSON.stringify(
+                { config: config.layerId, errors: validate.errors },
+                null,
+                2
+            )
+        )
+    }
+    return true
+}
+
+describe.each(earthEngineLayers)('GEE config validation', (config) => {
+    test('config is well-formed', () => {
```

Comment:
To be fancy, you could use a string literal and add the layer name into the test name.